doc/mysqlnd_azure_log.md

```diff
@@ -0,0 +1,124 @@
+# Diagnostic Log Configuration
+
+Log down some runtime information from mysqlnd\_azure.
+
+## Configuration
+
+### Log destination supported (mysqlnd\_azure.logOutput)
+
+**Notice:**
+For all the file paths mentioned below, except Apache and php-fpm's system error log, it need to be configured with right user permission, e.g. www-data for Apache. For relative file path, when working in cli, the father path is the current working directoy, when working with Apache, the father path is /.
+
+We provide multiple options for log destination you may want to choose. 
+
+Name |mysqlnd\_azure.logOutput
```

Comment:
This table format is confusing  , can we simplify this as 

 mysqlnd_azure.logOutput= 0 

0 is the default value but it can accept the values 1,2 and 4.  See details below for each value and choose the one based on your needs.  

Now show the Value/Behavior Table at line 23